e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/pdc-reconcile/mark-report-as-done.spec.js

```diff
@@ -1,33 +1,25 @@
-import { UTILISATION_REPORT_RECONCILIATION_STATUS, UtilisationReportEntityMockBuilder } from '@ukef/dtfs2-common';
+import {
+  UTILISATION_REPORT_RECONCILIATION_STATUS,
+  UtilisationReportEntityMockBuilder,
+  getReportPeriodForBankScheduleBySubmissionMonth,
+} from '@ukef/dtfs2-common';
 import pages from '../../../pages';
 import USERS from '../../../../fixtures/users';
 import { PDC_TEAMS } from '../../../../fixtures/teams';
 import { getMonthlyReportPeriodFromIsoSubmissionMonth, toIsoMonthStamp } from '../../../../support/utils/dateHelpers';
 import { NODE_TASKS } from '../../../../../../e2e-fixtures';
 import { aliasSelector } from '../../../../../../support/alias-selector';
+import { BANK1_PAYMENT_REPORT_OFFICER1 } from '../../../../../../e2e-fixtures/portal-users.fixture';
 
 context(`${PDC_TEAMS.PDC_RECONCILE} users can mark reports as done and not done`, () => {
+  let uploadedByUserId;
+
   const submissionMonth = toIsoMonthStamp(new Date());
-  const reportPeriod = getMonthlyReportPeriodFromIsoSubmissionMonth(submissionMonth);
   const utilisationReportsAlias = 'utilisationReportsAlias';
 
   const displayStatusSelector = 'td > strong[data-cy="utilisation-report-reconciliation-status"]';
-  const tableCellCheckboxSelector = (reportId) => `th > div > div > input[data-cy="table-cell-checkbox--set-status--reportId-${reportId}"]`;
-
-  const statusWithBankId = [
-    {
-      bankId: undefined,
-      status: UTILISATION_REPORT_RECONCILIATION_STATUS.REPORT_NOT_RECEIVED,
-    },
-    {
-      bankId: undefined,
-      status: UTILISATION_REPORT_RECONCILIATION_STATUS.PENDING_RECONCILIATION,
-    },
-    {
-      bankId: undefined,
-      status: UTILISATION_REPORT_RECONCILIATION_STATUS.RECONCILIATION_COMPLETED,
-    },
-  ];
+  const tableCellCheckboxSelector = (reportId, status) =>
+    `th > div > div > input[data-cy="table-cell-checkbox--set-status--reportId-${reportId}-currentStatus-${status}"]`;
```

Comment:
fwiw given the data-cy tag is so explicit I'm surprised to see that before we had a th >div > div selector rather than just on a data-cy check but doesn't really matter either way just giving some thoughts :D it makes the selector a little more brittle than just having a data-cy selector that should always be unique